specs/saltpack_armor.md

```diff
@@ -255,31 +255,46 @@ ipsum](https://twitter.com/oconnor663/status/680171387353448448).
 
 ## Framing the BaseX Payload
 
-Before getting to the BaseX payload, the decoder parses the header and footer:
+The BaseX payload is surrounded ('framed') by a header and footer. Rules for parsing
+them appear here.
 
-1. Strip all whitespace and `>` characters from the message. (`>` is for
-   compatibility with email clients that use it for quoting.)
-2. Collect input up to the first period. This is the header.
-3. Assert that the header (with its whitespace stripped in step 1) matches
-
-   ```
-   BEGIN([a-zA-Z0-9]+)?SALTPACK(ENCRYPTEDMESSAGE)|(SIGNEDMESSAGE)|(DETACHEDSIGNATURE)
-   ```
-
-   The optional word is for an application name (like `KEYBASE`). The last two
-   words give the mode of the message.
-4. Collect input up to the second period. This is the payload. If the
-   implementation is streaming, it may decode the payload before the following
-   steps.
+0. For this section, consider the "stripping set" to be the union of all whitespace and
```

Comment:
I think markdown will actually render `0.` as `1.`. In general it ignores the actual numbers you use.

Also, specify whether you mean ASCII whitespace or all Unicode whitespace.